### ISSUE TITLE
Support mounting an EFS volume to EC2 instances the same way as EBS volumes

### ISSUE THREAD

**Issue by tomcook** [cloudwishlist/aws#7]
The EC2 API allows for mounting EBS volumes to specific mount points at instance creation time or at any other time during the life of the instance. EFS volumes, however, are a purely OS-level mount, that have to be mounted once the instance is running.

EC2 should support virtually mounting EFS volumes in a similar scheme as EBS volumes, so that instances automatically come up with EFS mounts in place, or can have them added later without running OS-level NFS commands.

This would simplify instance configuration and eliminate the dependency on NFS as well.

**Comment by jmason888** [comment]
Agreed.  As a workaround, I'm planning to use small shell scripts run by SSM documents for that purpose.
